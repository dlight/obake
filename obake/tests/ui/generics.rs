#[obake::versioned]
struct Foo<T> {}

#[obake::versioned]
struct Bar<'a> {}

#[obake::versioned]
enum Baz<T> {}

#[obake::versioned]
enum Flim<'a> {}

fn main() {}
