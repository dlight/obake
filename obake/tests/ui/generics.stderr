error: expected curly braces
 --> $DIR/generics.rs:2:11
  |
2 | struct Foo<T> {}
  |           ^

error: expected curly braces
 --> $DIR/generics.rs:5:11
  |
5 | struct Bar<'a> {}
  |           ^

error: expected curly braces
 --> $DIR/generics.rs:8:9
  |
8 | enum Baz<T> {}
  |         ^

error: expected curly braces
  --> $DIR/generics.rs:11:10
   |
11 | enum Flim<'a> {}
   |          ^
